public/html/editmode.html

```diff
@@ -0,0 +1,312 @@
+                margin: 10px;
+                background-color: white;
+            }
+
+        </style>
+    </head>
+    <body>
+        <div id="toolBarDiv">
+            
+            <div id="slideTools" class="toolBarCont">
+                <a href="#">+</a>
+            </div>
+
+            <div id="items" class="toolBarCont">
+                <button>Text Field</button>
+                <button>List</button>
+                <input type="file">
+            </div>
+
+            <div id="textEditTools" class="toolBarCont">
+                <select id="fontFamSelect">
+                    <option>Calibri</option>
+                    <option>Helvetica</option>
+                    <option>Times New Roman</option>
+                </select>
+
+                <select id="fontSizeSelect">
+                    <option>12 pt</option>
+                    <option>14 pt</option>
+                    <option>18 pt</option>
+                    <option>24 pt</option>
+                    <option>36 pt</option>
+                    <option>48 pt</option>
+                </select>
+
+                <button class="nbi" style="text-decoration: underline">U</button>
+                <button class="nbi" style="font-weight: bold">B</button>
+                <button class="nbi" style="font-style:italic">I</button>
+
+                <label for="textColorInp">Text Color</label>
+                <input id="textColorInp" type="color" style="width: 50px">
+
+                <button class="lcr" style="border-left: 1px solid black;">L</button>
+                <button class="lcr" style="border-bottom: 1px solid black;">C</button>
+                <button class="lcr" style="border-right: 1px solid black;">R</button>
+            </div>
+
+            <div id="typeOfSlides" class="toolBarCont">
+                <div>Title</div>
+                <div>Text and Image</div>
+                <div>List</div>
+            </div>
+
+            <div id="setSlidebackgroundDiv" class="toolBarCont">
+                    <label for="backgroundColorInp">Background Color</label>
+                    <input id="backgroundColorInp" type="color" style="width: 50px">
+                    <br>
+                    <label for="backgroundImgInp">Image</label>
+                    <input id="backgroundImgInp" type="file">
+            </div>
+
+            <div id="navBtnsCont" class="toolBarCont"> 
+                <button id="saveBtn" style="background-color: #2f71e3; border-color: #2f71e3">Save</button>
+                <button id="backBtn">Back</button>
+            </div>
+            
+        </div>
+
+        <div id="underToolBarDiv">
+
+            <div id="allSlidesPreviewCont">
+                <!-- Here goes all the slidepreviews, JavaScript -->   
+            </div>
+
+            <div id="editSlideCont">
+                <!-- Here goes the code to edit a slide, JavaScript -->
+            </div>
+
+        </div>
+    </body>
+    <script>
+
```

Comment:
If it's javascript it could be in it's own file. It would greatly improve readability since this page needs to communicate with the server to fetch the presentation it would get big really fast